projects/kubernetes/kubernetes/README.md

```diff
@@ -24,8 +24,8 @@ If you are building from macOS, and have installed go 1.15.3 with
 $ go env GOROOT
 /usr/local/Cellar/go/1.15.3/libexec
 ```
-In our [build container](../../../builder-base/README.md), you can see we
-get Go from https://golang.org/dl/] and install it in `/usr/local`
```

Comment:
Not fixed and link is still broken. I'll make this the only change in this PR for the next revision

Reply:
Never mind. Opening new PR.